Floofbot/Migrations/20200503231617_AlterTagTable.cs

```diff
@@ -6,14 +6,25 @@ public partial class AlterTagTable : Migration
     {
         protected override void Up(MigrationBuilder migrationBuilder)
         {
+            migrationBuilder.AddColumn<ulong>(
+                name: "ServerId",
+                table: "Tags",
+                nullable: false,
+                defaultValue: 0ul);
```

Comment:
default value 0ul?

Reply:
It means 0, but specifically unsigned long.